scripts/release/publish-commands/validate-tag.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const {readJson} = require('fs-extra');
+const {join} = require('path');
+const theme = require('../theme');
+
+const run = async ({cwd, packages, tag}) => {
+  // Prevent a "next" release from ever being published as @latest
+  // All canaries share a version number, so it's okay to check any of them.
+  const arbitraryPackageName = packages[0];
+  const packageJSONPath = join(
+    cwd,
+    'build',
+    'node_modules',
+    arbitraryPackageName,
+    'package.json'
+  );
+  const {version} = await readJson(packageJSONPath);
+  if (version.indexOf('0.0.0') === 0) {
+    if (tag === 'latest') {
+      if (version.indexOf('experimental') !== -1) {
```

Comment:
Looks like you meant to use `===` here? (Or you meant to reverse the log messages below?)

Given the number of `indexOf` checks in this section, it might be clearer if you assigned that to a variable e.g. `isExperimentalVersion`.